localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```diff
@@ -56,12 +56,6 @@
  innovation_hexer:0 "Hexers"
  innovation_hexer_desc:0 "The hexers hold a highly important position in our society, both in terms of culture and defense. They act as leaders to our people, both temporally and spiritually. Furthermore, their magic enables them to sap an opponent's strength, as well as invoke the spirits in our favor."
 
- innovation_marauder:0 "Marauders"
- innovation_marauder_desc:0 "Like most of their kin, Marauders are savagery and brutality incarnate. Marauders are fast, aggressive and keen to show no mercy. Donning the hides of their enemies, Marauders are not known for taking prisoners."
-
- innovation_magi:0 "Magi"
- innovation_magi_desc:0 "Ogre Magi are skilled spellcasting warriors. The two-headed ogre spellcasters are typically larger and more intelligent than their one-headed brethren. They often enjoy an elevated position among the ogre clans."
-
```

Comment:
When we remove lines, we leave empty lines so lines below keep their line number.